Sources/BarcodeScannerController.swift

```diff
@@ -449,3 +449,26 @@ extension BarcodeScannerController: HeaderViewDelegate {
     dismissalDelegate?.barcodeScannerDidDismiss(self)
   }
 }
+
+extension AVCaptureVideoOrientation {
```

Comment:
I would move it to a separate file, there are a lot of code in the controller already.